tests/test_bukuDb.py

```diff
@@ -1318,19 +1315,15 @@ def test_load_chrome_database(chrome_db, add_pt):
 
 @pytest.fixture()
 def firefox_db(tmpdir):
-    zip_url = "https://github.com/jarun/buku/files/1319933/bookmarks.zip"
```

Comment:
…Let's maybe not share 10Mb of your personal browsing history with anyone who wants to run tests, shall we :sweat_smile: